products/bash/release-mq.sh

```diff
@@ -17,6 +17,7 @@
 #   -r : <release_name> (string), Defaults to "mq-demo"
 #   -i : <image_name> (string)
 #   -q : <qm_name> (string), Defaults to "QUICKSTART"
+#   -z : <tracing_namespace> (string), Defaults to ""
```

Comment:
Doc that "" means use same namespace as -n

Reply:
fixed